test/integration/expression-tests/diacritics/locale/test.json

```diff
@@ -1,5 +1,5 @@
 {
-  "expression": ["diacritic-insensitive-equal", "de", ["get", "lhs"], ["get", "rhs"]],
+  "expression": ["==", ["string",["get", "lhs"]], ["get", "rhs"], ["collator", true, false, "de"]],
```

Comment:
Maybe we should consider accepting an single object literal for collator options, e.g. `["collator", { caseSensitive: false, diacriticSensitive: false, locale: 'de' }]`.  It's not how any existing expressions work, but this is the first case where we really need a set of options, and a dictionary is much clearer than a positional arguments for this.

Reply:
And while we're considering such a departure, we might as well ask: should we simply accept a collator options object directly as an optional fourth argument to collating expressions?  

```js
["==", ["string",["get", "lhs"]], ["get", "rhs"], { caseSensitive: false, diacriticSensitive: false, locale: 'de' }]
```